Slight clarify in optimization tutorial (#5284)
godotengine/godot-docs: tutorials/performance/cpu_optimization.rst

Although Nodes are an incredibly powerful and versatile concept, be aware that
every node has a cost. Built-in functions such as `_process()` and

> `_physics_process()` propagate through the tree. This housekeeping can reduce 
> performance when you have a very large numbers of nodes (how many exactly 
> depends on the target platform and can range from thousands to tens of 
> thousands so ensure that you profile performance on all target platforms 
> during development).

Reviewer: ```suggestion
`_physics_process()` propagate through the tree. This housekeeping can reduce 
performance when you have a very large numbers of nodes (how many exactly 
depends on the target platform and other factors).
```

Although Nodes are an incredibly powerful and versatile concept, be aware that
every node has a cost. Built-in functions such as `_process()` and
`_physics_process()` propagate through the tree. This housekeeping does start to reduce

> performance when you have very large numbers of nodes (depending on platform, but about 10% of frame time per ten thousand nodes on a desktop CPU).

Reviewer 2: I'm not sure if giving an "accurate" figure is a good idea here. People may start extrapolating based on this in weird ways, especially since the CPU model isn't mentioned.

Author: Mmmm, agreed. I however have lived for two years worrying about having 1000 nodes in my tree, which as it seems is completely unnecessary. So it may also lead to people trying to over optimize causing slowdown in production time. I do get your point and agree with you when I think about it, perhaps "does start to" is better than "can" but other than that we can close this PR
